CITATION.cff

```diff
@@ -1,59 +1,63 @@
-  given-names: "A."
-- family-names: "Vélez-Fort"
-  given-names: "M."
-- family-names: "Rousseau"
-  given-names: "C."
-- family-names: "Cossell"
-  given-names: "L."
-- family-names: "Tsitoura"
-  given-names: " C."
-- family-names: "Lenzi"
-  given-names: " S."
-- family-names: "Obenhaus"
-  given-names: "H."
-- family-names: "Claudi"
-  given-names: "F."
-- family-names: "Branco"
-  given-names: "T."
-- family-names: "Margrie"
-  given-names: "T."
-url: https://github.com/brainglobe/brainreg
+  - given-names: Adam
+    family-names: Tyson
+    email: hello@brainglobe.info
+    affiliation: 'Sainsbury Wellcome Centre, University College London'
+  - given-names: Charly
+    family-names: Rousseau
+  - given-names: Stephen
+    family-names: Lenzi
+  - given-names: BrainGlobe
+    family-names: Developers
+    email: hello@brainglobe.info
+repository-code: 'https://github.com/brainglobe/brainreg'
+url: 'https://brainglobe.info'
+abstract: Automated multi-atlas whole-brain microscopy registration.
+license: BSD-3-Clause
+date-released: '2024-01-25'
+year: 2024
 preferred-citation:
-    type: article
-    title: Accurate determination of marker location within whole-brain microscopy images
-    journal: Scientific Reports
-    volume: 12
-    pages: 867
-    doi: 10.1038/s41598-021-04676-9
-    year: 2022
-    date-published: 2022-01-18
-    authors:
-      - family-names: "Tyson"
-        given-names: "A."
-      - family-names: "Vélez-Fort"
-        given-names: "M."
-      - family-names: "Rousseau"
-        given-names: "C."
-      - family-names: "Cossell"
-        given-names: "L."
-      - family-names: "Tsitoura"
-        given-names: " C."
-      - family-names: "Lenzi"
-        given-names: "S."
-      - family-names: "Obenhaus"
-        given-names: "H."
-      - family-names: "Claudi"
-        given-names: "F."
-      - family-names: "Branco"
-        given-names: "T."
-      - family-names: "Margrie"
-        given-names: "T."
+  type: article
+  authors:
+  - given-names: Adam
+    family-names: Tyson
+    affiliation: 'Sainsbury Wellcome Centre, University College London'
+  - given-names: Mateo
+    family-names: Velez-Fort
+  - given-names: Charly V.
+    family-names: Rousseau
+  - given-names: Lee
+    family-names: Cossell
+  - given-names: Chryssanthi
+    family-names: Tsitoura
+  - given-names: Stephen C.
+    family-names: Lenzi
+  - given-names: Horst A.
+    family-names: Obenhaus
+  - given-names: Federico
+    family-names: Claudi
+  - given-names: Tiago
+    family-names: Branco
+  - given-names: Troy W.
+    family-names: Margrie
+  doi: "10.1038/s41598-021-04676-9"
+  url: "https://doi.org/10.1038/s41598-021-04676-9"
+  journal: "Scientific Reports"
+  month: 1
+  year: 2022
+  title: "Accurate determination of marker location within whole-brain microscopy images"
+  issue: 1
+  volume: 12
+citation-sentence: "This project uses the brainreg pipeline, an update to aMAP, to register template brains to atlases."
```

Comment:
I can't come up with a good single sentance that could be used for all projects, so I suggest scrapping for now, and think later how this should  best cited. 
```suggestion
```